src/messaging/ReliableMessageMgr.h

```diff
@@ -206,6 +206,14 @@ class ReliableMessageMgr
 #endif // CHIP_CONFIG_TEST
 
 private:
+    /**
+     * Calculates the next retransmission time for the entry
+     * Function sets the nextRetransTime of the entry
+     *
+     * @param[in/out] entry RetransTableEntry for which we need to calculate the nextRetransTime
+     */
+    void CalculateNextRetransTime(RetransTableEntry * entry);
```

Comment:
Please use a ref instead of a pointer, since it should never be null.

Reply:
done